hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieLogFile.java

```diff
@@ -104,7 +117,7 @@ public HoodieLogFile rollOver(FileSystem fs) throws IOException {
       }
       // reverse the order by base-commits
       return new Integer(baseInstantTime2.compareTo(baseInstantTime1));
-    };
+    }
   }
 
   @Override
```

Comment:
Let java re-create the hashcode and equals based on all the new members in the object ?